platform/src/main/java/org/stellar/anchor/platform/controller/Sep24Controller.java

```diff
@@ -0,0 +1,208 @@
+package org.stellar.anchor.platform.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.*;
+import org.stellar.anchor.dto.SepExceptionResponse;
+import org.stellar.anchor.dto.sep24.*;
+import org.stellar.anchor.exception.SepException;
+import org.stellar.anchor.exception.SepNotFoundException;
+import org.stellar.anchor.exception.SepValidationException;
+import org.stellar.anchor.sep10.JwtToken;
+import org.stellar.anchor.sep24.Sep24Service;
+
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.stellar.anchor.filter.BaseTokenFilter.JWT_TOKEN;
+import static org.stellar.anchor.util.Log.*;
+
+@RestController
+@CrossOrigin(origins = "*")
+@RequestMapping("/sep24")
+public class Sep24Controller {
+  private final Sep24Service sep24Service;
+
+  Sep24Controller(Sep24Service sep24Service) {
+    this.sep24Service = sep24Service;
+  }
+
+  @CrossOrigin(origins = "*")
+  @RequestMapping(
+      value = "/info",
+      method = {RequestMethod.GET})
+  public InfoResponse getInfo() {
+    debug("/info");
+    return sep24Service.getInfo();
+  }
+
+  @CrossOrigin(origins = "*")
+  @RequestMapping(
+      value = "/transactions/deposit/interactive",
+      consumes = {MediaType.APPLICATION_JSON_VALUE},
+      method = {RequestMethod.POST})
+  public InteractiveTransactionResponse deposit(
+      HttpServletRequest request, @RequestBody HashMap<String, String> requestData)
+      throws SepException, MalformedURLException, URISyntaxException {
+    debug("/deposit", requestData);
```

Comment:
👀  this could be actually leaking PII information, right?

Reply:
Yes, this will go to the debug log. The anchor will need to take care of the log files carefully. 

The PIIs should be ok as long as the anchor do not ship them out. If they would like to ship the log files outside, they will need to apply redaction.

Reply:
Ok, we just need to make sure the recommended/safe log level configuration is INFO and not DEBUG then.